tests/B2CWebAppCallsWebApi/Client/Services/TodoListService.cs

```diff
@@ -144,5 +146,20 @@ public async Task<Todo> GetAsync(int id)
 
             throw new HttpRequestException($"Invalid status code in the HttpResponseMessage: {response.StatusCode}.");
         }
+
+        private async Task<HttpRequestMessage> PrepareAuthenticatedClient(
```

Comment:
>PrepareAuthenticatedClient [](start = 47, length = 26)

nit: Can we find a place to share this code across test projects? perhaps by using a shared project?

Reply:
I would wait a bit for that. I would like our test projects to start leveraging (and extending) the unfolded templates: See https://github.com/AzureAD/microsoft-identity-web/issues/355